### PR TITLE
Adds emulated test comments

### PR DESCRIPTION
**Pull request by claravanstaden**
Adds comments for the Snowbridge emulated tests.

Snowbridge companion: https://github.com/Snowfork/snowbridge/pull/1110

### CODE REVIEW

File: cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs [hunk 1]
```diff
@@ -48,17 +48,19 @@ pub enum SnowbridgeControl {
 	Control(ControlCall),
 }
 
+/// Create an agent on Ethereum. An agent is a representation of an entity in the Polkadot
+/// ecosystem (like a parachain) on Ethereum.
```
Comment (vgeddes) [inline cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:52]: I don't think rust-doc `///` comments are meaningful for tests. Should probably just use `//`

```suggestion
// Create an agent on Ethereum. An agent is a remote proxy for a consensus system within Polkadot 
```

File: cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs [hunk 2]
```diff
@@ -102,18 +104,19 @@ fn create_agent() {
 	});
 }
 
+/// Create a channel for an agent. A channel is like a lane, specifically used by one agent.
```
Comment (vgeddes) [inline cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:107]:
```suggestion
// Create a channel for a consensus system. A channel is a bidirectional messaging channel between BridgeHub and Ethereum.
```

File: cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs [hunk 3]
```diff
@@ -90,7 +92,7 @@ fn create_agent() {
 
 	BridgeHubRococo::execute_with(|| {
 		type RuntimeEvent = <BridgeHubRococo as Chain>::RuntimeEvent;
-
+		// Check that the Agent was created
```
Comment (vgeddes) [inline cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:95]:
```suggestion
		// Check that a message was sent to Ethereum to create the agent
```

File: cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs [hunk 4]
```diff
@@ -102,18 +104,19 @@ fn create_agent() {
...
+	// Fund AssetHub sovereign account so that it can pay execution fees.
 	BridgeHubRococo::fund_para_sovereign(origin_para.into(), INITIAL_FUND);
 
 	let sudo_origin = <Rococo as Chain>::RuntimeOrigin::root();
 	let destination: VersionedMultiLocation =
 		Rococo::child_location_of(BridgeHubRococo::para_id()).into();
 
 	let create_agent_call = SnowbridgeControl::Control(ControlCall::CreateAgent {});
-
+	// Create Channel XCM call
```
Comment (vgeddes) [inline cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:119]:
```suggestion
	// Construct XCMs to create an agent and a channel for para 1001
```

File: cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs [hunk 5]
```diff
@@ -102,18 +104,19 @@ fn create_agent() {
 	});
 }
 
+/// Create a channel for an agent. A channel is like a lane, specifically used by one agent.
 #[test]
 fn create_channel() {
 	let origin_para: u32 = 1001;
-
+	// Fund AssetHub sovereign account so that it can pay execution fees.
```
Comment (vgeddes) [inline cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:111]: `origin_para` (Para 1001) isn't AssetHub.

File: cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs (hunk 1 above)
Comment (claravanstaden) [reply cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:52]: I thought the same but then I checked what Parity does, and it looks like they use `///` for test comments fairly consistently:

- https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/client/consensus/common/src/tests.rs#L1274
- https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/pallets/dmp-queue/src/tests.rs#L153
- https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/pallets/xcmp-queue/src/tests.rs#L117

Comment (vgeddes) [reply cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs:52]: ah ok, :+1:

**Review by vgeddes — APPROVED** [review]

**Pull request merged**